doc/dev/crimson/pipelines.rst

```diff
@@ -0,0 +1,279 @@
+=========
+Pipelines
+=========
+
+RADOS Protocol Requirements
+===========================
+
+A Ready-to-Go Checklist
+-----------------------
+
+In Ceph, librados I/O requests inquiry or mutate the shared state of the cluster.
+Because inherently a distributed storage system is distributed by its own nature, we can not assume
+that an OSD or the PGs hosted by it are always ready to serve a request targeting a shared state, not
```

Comment:
s/inherently//
s/its own//

Not sure what "targeting a shared state" is about, so suggest removing those words too.